system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -42,6 +43,54 @@
                            (map #(select-keys % [:concept-id :extra-fields :metadata])))]
     (#'jobs/process-subscriptions (system/context) subscriptions)))
 
+
+(deftest subscription-ingest-no-subscriber-id-test
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-no-id"
+                 :EntryTitle "entry-title-no-id"})
+               {:token "mock-echo-system-token"})]
+    (testing "ingest on PROV1, with no subscriber-id supplied"
+      (let [concept (subscription-util/make-subscription-concept {:provider-id "PROV3"
+                                                                  :CollectionConceptId (:concept-id coll1)
+                                                                  :SubscriberId nil})
+            user1-token (echo-util/login (system/context) "user1")
+            response (ingest/ingest-concept concept {:token user1-token})
+            ingested-concept (mdb/get-concept (:concept-id response))
+            parsed-metadata (json/parse-string (:metadata ingested-concept) true)]
+        (is (= 201 (:status response)))
+        (is (= "user1" (:SubscriberId parsed-metadata)))))))
+
+(deftest subscription-ingest-no-email-test
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-no-id"
+                 :EntryTitle "entry-title-no-id"})
+               {:token "mock-echo-system-token"})]
+    (testing "ingest on PROV1, with no email or subscriber id supplied"
```

Comment:
For completeness, add test cases with blank subscriber-id and email provided.

Reply:
The above "no subscriber id" test actually was already doing this, but I've gone ahead and made it more explicit.

Reply:
by `blank`, I mean `         `. I think this should trigger a validation error rather than applying the default.

Reply:
This validation issue will be investigated and addressed in by CMR-7169.